grade-scripts/shared/utils.js

```diff
@@ -13,6 +15,53 @@ const responseIsOK = (response, siteToCheck) => {
         firstStatusDigit === 3
 }
 
+const getSiteData = (inputPath, fileForSubset) => {
+    // get initial file data
+    let siteDataFiles = fs.readdirSync(inputPath)
```

Comment:
What do you think about validating the inputPath via [fs.existsSync(path)](https://nodejs.org/api/fs.html#fs_fs_existssync_path) and failing gracefully if necessary?
If inputPath is undefined or invalid, right now it will throw an invalid directory stream descriptor error. See [readdirSync](https://nodejs.org/api/fs.html#fs_fs_readdirsync_path_options) reference and in particular the [readdir](http://man7.org/linux/man-pages/man3/readdir.3.html) man-page